.github/workflows/i18n.yml

```diff
@@ -1,63 +1,46 @@
-name: Locale Updater
+name: Update locale files
 
 on:
   push:
     branches:
-      - master
       - main
+      - master
+
+permissions:
+  pull-requests: write
+  contents: write
 
 jobs:
   locale-updater:
+    if: github.repository == 'jazzband/djangorestframework-simplejwt'
     name: Locale updater
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
     steps:
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      with:
-        ref: ${{ github.head_ref }}
 
-    - name: Set up Python 3.9
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.9'
+    - name: Checkout Repository
+      uses: actions/checkout@v4
 
-    - name: Get pip cache dir
-      id: pip-cache
-      run: |
-        echo "::set-output name=dir::$(pip cache dir)"
-
-    - name: Cache
-      uses: actions/cache@v2
+    - name: Set up Python
+      uses: actions/setup-python@v5
       with:
-        path: ${{ steps.pip-cache.outputs.dir }}
-        key:
-          3.9-v1-${{ hashFiles('**/setup.py') }}
-        restore-keys: |
-          3.9-v1-
+        python-version: '3.12'
+        cache: 'pip'
+        cache-dependency-path: setup.py
 
     - name: Install dependencies
       run: |
         sudo apt-get install -y gettext 
         python -m pip install --upgrade pip wheel setuptools
         pip install -e .[dev]
 
-    - name: Run locale
-      working-directory: rest_framework_simplejwt
```

Comment:
Did you test this change? I don't know why it was added but could it effect the script run? I didn't actually look at the script.